src/upgrade/UpgradeCard.options.js

```diff
@@ -0,0 +1,389 @@
+export const getOptions = () => {
+  const options = {
```

Comment:
@nazaninreihani I am not sure if we have to export a function. Can we not just do `export default { all your options data }`?